pkg/kubelet/config/file_linux_test.go

```diff
@@ -374,10 +371,9 @@ func expectUpdate(t *testing.T, ch chan interface{}, testCase *testCase) {
 				}
 			}
 
-			testCase.lock.Lock()
-			defer testCase.lock.Unlock()
```

Comment:
why is it safe to remove the lock? we're still checking against testcase.expected

Reply:
It is safe because only one goroutine uses `testCase` now.  `go changeFile()` does not modify `testCase` anymore(we didn't have to).